pulp_ansible/app/utils.py

```diff
@@ -26,3 +27,24 @@ def filter_content_for_repo_version(qs, repo_version):
     content_rel = RepositoryContent.objects.filter(f)
 
     return qs.filter(pk__in=content_rel.values_list("content_id"))
+
+
+def get_queryset_annotated_with_task_sync_task(qs):
+    tasks = (
+        Task.objects.filter(
+            name__contains="sync",
+            reserved_resources_record__icontains=Cast(OuterRef("pk"), output_field=CharField()),
+        )
+        .values(
+            json=JSONObject(
+                pk="pk",
+                state="state",
+                pulp_created="pulp_created",
+                finished_at="finished_at",
+                error="error",
+            )
```

Comment:
Can you remove this JSONObject and just have the values be the fields?

Reply:
Well, without using `JSONObject`, I get this
```
Exception Value: | subquery must return only one column LINE 1: ...yversion"."complete") AS "latest_version_number", (SELECT U0...                                                              ^
```